code/modules/clothing/outfits/standard.dm

```diff
@@ -333,6 +333,8 @@
 	r_pocket = /obj/item/teleportation_scroll/apprentice
 	r_hand = null
 	l_hand = null
+	backpack_contents = list(/obj/item/storage/box/survival = 1,
+)
```

Comment:
why not just make backpack contents 1 line

Reply:
because dweebs for a lazy and doesn't want to press enter to move the thing downwards when they add objects